A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -774,6 +775,21 @@ var A320_Neo_UpperECAM;
                     }
                 }
             }
+            var currentThrottleState = Math.max(Simplane.getEngineThrottleMode(0), Simplane.getEngineThrottleMode(1));
+            if(this.throttleMode != currentThrottleState){
+                this.throttleMode = currentThrottleState;
+            }
```

Comment:
what's the point of this?

Reply:
So the N1 and EGT limits vary based on engine modes and for TOGA there's supposedly a 5min limit on the max limit and then it falls back to MCT limit, this'll have to hold the previous throttle state to check if its been TOGA or changed to TOGA so that the timer maybe reset to 5min(300s).

Reply:
you're not doing anything in that if statement though. If you wanted to do some right when the state actually changed, it would have to go inside this if statement right?

Reply:
its updating the stored value for throttle state, that is its function.

Reply:
in that case, no need for the if, just set it every frame, it won't hurt anything